tests/ignite/handlers/test_param_scheduler.py

```diff
@@ -637,66 +632,69 @@ def test_lr_scheduler_asserts():
         LRScheduler.simulate_values(1, None)
 
 
-def test_lr_scheduler():
-    def _test(torch_lr_scheduler_cls, **kwargs):
+@pytest.mark.parametrize(
+    "torch_lr_scheduler_cls, kwargs",
+    [
+        (StepLR, ({"step_size": 5, "gamma": 0.5})),
+        (ExponentialLR, ({"gamma": 0.78})),
+        (MultiplicativeLR, ({"lr_lambda": lambda epoch: 0.95})),
```

Comment:
This wont work as older versions wont be able to import the class

Reply:
I have added a fix, please have a look thanks!